app/exporter/logic/exports.py

```diff
@@ -0,0 +1,85 @@
+"""
+    This module contains all business logic around exports, including
+    validating ownership, creating new models, and processing pending
+    exports
+"""
+import os
+import csv
+from uuid import uuid4
+
+from exporter.db import exports
+from exporter.models.export import Export
+from exporter.logic import mailgun
+from exporter.logic import s3
+from flask import abort
+
+
+def get(user, id):
+    export = exports.find_one(Export.key(user, id))
+    return Export(export)
+
+
+def list_exports(user, domain):
+    export_list = exports.find({
+        'domain': domain,
+        'user': user.key
+    })
+    return [Export(e) for e in export_list]
+
+
+def create(user, domain, export_type):
+
+    if export_type not in Export.TYPES:
+        abort(400)
+
+    new_export = Export({
+        "domain": domain,
+        "user": user.key,
```

Comment:
So, yeah, I am. If you look at the file history, it was originally stored as a one way hash. But because I have to run jobs later, I have to store the key somewhere. I could store it on the export until the export has run, but that's no better. 

The best answer I feel in to encrypt it using AES or similar. Thoughts?